modin/experimental/core/execution/native/implementations/omnisci_on_native/dataframe/dataframe.py

```diff
@@ -267,23 +267,23 @@ def ref(self, col):
 
     def mask(
         self,
-        row_indices=None,
-        row_numeric_idx=None,
-        col_indices=None,
-        col_numeric_idx=None,
+        row_labels=None,
+        row_positions=None,
+        col_labels=None,
+        col_positions=None,
```

Comment:
Why we don't add the type hints here as it is already done for `ModinDataframe`/`PandasDataFrame`?